blackbox/bootstrap.sh

```diff
@@ -1,14 +1,25 @@
 #!/bin/bash
 
 if hash python3.6 2> /dev/null; then
-    python3.6 -m venv .venv
+    PYTHON=python3.6 
 elif hash python3 2> /dev/null; then
     # fallback to python3 in case there is no python3.6 alias; should be 3.6
-    python3 -m venv .venv
+    PYTHON=python3
 else
     echo 'python3.6 required'
     exit 1
 fi
 
+case $OSTYPE in
```

Comment:
Actually I would change the condition here and don't check for the **OS** but instead check if the `./venv/bin/pip` exists and if not then download it manually with wget.

Reply:
Reason is that this problem hasn't occured to us so far and seems to be related to the Anaconda-python installation and not to macOS.